### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by achingbrain**
BREAKING CHANGE: uses new multiaddr, libp2p-interfaces, etc

### CODE REVIEW

File: test/floodsub.spec.js [hunk 1]
```diff
@@ -4,107 +4,113 @@
...
+        receivedFrom: floodsub.peerId.toB58String(),
         data: message,
         topicIDs: [topic]
       }))
 
     expect(messageToEmit).to.eql(expected)
   })
+
+  it.skip('does not send received message back to original sender', async () => {
+
```
Comment (vasco-santos) [inline test/floodsub.spec.js:114]: Needs test?

File: test/floodsub.spec.js [hunk 2]
```diff
@@ -4,107 +4,113 @@
...
+  let floodsub
 
   before(async () => {
     expect(Floodsub.multicodec).to.exist()
 
-    ;[peer1, peer2] = await createPeers({ number: 2 })
-    floodsub1 = new Floodsub(peer1, defOptions)
-    floodsub2 = new Floodsub(peer2, defOptions)
+    const libp2p = {
+      peerId: await PeerId.create(),
```
Comment (vasco-santos) [inline test/floodsub.spec.js:31]: any reason to not use createPeer?

File: test/floodsub.spec.js (hunk 1 above)
Comment (achingbrain) [reply test/floodsub.spec.js:114]: Yeah, just writing it.  I pushed an update to see if CI was healthier with the libp2p rc, it is!

File: test/floodsub.spec.js (hunk 2 above)
Comment (achingbrain) [reply test/floodsub.spec.js:31]: Spinning up libp2p nodes is overkill for unit tests.  Ideally we wouldn't depend on libp2p at all in this module as libp2p itself depends on floodsub creating a loop which we really need to break in order to simplify publishing new versions.

File: test/floodsub.spec.js (hunk 1 above)
Comment (achingbrain) [reply test/floodsub.spec.js:115]: Added now.

**Review by vasco-santos — APPROVED** [review]
LFTM

**Pull request merged**